.github/workflows/android.yml

```diff
@@ -16,16 +16,31 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
```

Comment:
I think you could simply do:

```
matrix:
  all_files_access: ['OFF', 'ON']
  include:
    - triplet: ...
      qt_arch: ...
    - ...
```

80% chance it works

Reply:
@m-kuhn , I tried first, and it felt into the 20% not working :)